openapi/src/parts/controlled_aws_sagemaker_notebook.yaml

```diff
@@ -0,0 +1,232 @@
+      tags: [ ControlledAwsResource ]
+      responses:
+        '200':
+          $ref: '#/components/responses/CreatedControlledAwsSageMakerNotebookResponse'
+        '202':
+          $ref: '#/components/responses/CreatedControlledAwsSageMakerNotebookResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/sagemaker-notebooks/{resourceId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+    get:
+      summary: Get a controlled AWS SageMaker Notebook resource
+      operationId: getAwsSageMakerNotebook
+      tags: [ControlledAwsResource]
+      responses:
+        '200':
+          $ref: '#/components/responses/GetControlledAwsSageMakerNotebookResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/sagemaker-notebooks/{resourceId}/consoleLink:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+      - $ref: '#/components/parameters/AwsCredentialAccessScope'
+      - $ref: '#/components/parameters/AwsConsoleSessionDuration'
+    get:
+      summary: Get a link to manage this notebook in the AWS SageMaker Console
+      operationId: getAwsSageMakerNotebookConsoleLink
+      tags: [ ControlledAwsResource ]
+      responses:
+        '200':
+          $ref: '#/components/responses/GetControlledAwsSageMakerNotebookConsoleLinkResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/sagemaker-notebooks/{resourceId}/credential:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+      - $ref: '#/components/parameters/AwsCredentialAccessScope'
+      - $ref: '#/components/parameters/AwsCredentialDuration'
+    get:
+      summary: Get a credential to access this notebook instance in the AWS CLI or SDK
+      operationId: getAwsSageMakerNotebookCredential
+      tags: [ ControlledAwsResource ]
+      responses:
+        '200':
+          $ref: '#/components/responses/GetControlledAwsSageMakerNotebookCredentialResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/sagemaker-notebooks/{resourceId}/proxyUrl:
```

Comment:
Why doesn't the general GET method provide the proxyUrl?
Or is this another credential case? (see comment above)